core/model/modx/moduser.class.php

```diff
@@ -916,12 +916,17 @@ public function getProfilePhoto($width = 128, $height = 128) {
         $this->xpdo->loadClass('sources.modMediaSource');
         /** @var modMediaSource $source */
         $source = modMediaSource::getDefaultSource($this->xpdo, $this->xpdo->getOption('photo_profile_source'));
-        $source->initialize();
-
-        $path = $source->prepareSrcForThumb($this->Profile->photo);
```

Comment:
What's the reason for those lines to be deleted? Just curious.

Reply:
Because this line exists in phpthumb processor: https://github.com/modxcms/revolution/blob/3.x/core/model/modx/processors/system/phpthumb.class.php#L54